contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3749,8 +3802,10 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 					 */
 					if (pstmt->stmt_len == 0 && list_length(grant->privileges) == 0)
 					{
-						if(check_bbf_schema_for_schema(logicalschema, "ALL", "execute"))
+						if(check_bbf_schema_for_schema(logicalschema, "ALL", 32))
+						{
```

Comment:
32 means only for execute privilege bit has been set that means only execute privilege is granted.